starknet_py/tests/e2e/client/conftest.py

```diff
@@ -15,14 +15,14 @@
 
 directory = os.path.dirname(__file__)
 
+SCRIPT_PATH = Path(directory) / "prepare_devnet_for_gateway_test.sh"
+CONTRACT_COMPILED = directory_with_contracts / "balance_compiled.json"
+CONTRACT_ABI = directory_with_contracts / "balance_abi.json"
```

Comment:
I'm not sure if it makes sense to extract these as global constants if these values aren't reused in any other test.